package appium;

import java.io.IOException;

import org.testng.annotations.Test;

public class Apractice  {

	@Test
	private void testApractice() throws InterruptedException, IOException {

	}

}
